.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -7,6 +7,9 @@ assignees: ''
 
 ---
 
+**Xircuits Version**
```

Comment:
The entire folder `ISSUE_TEMPLATE` does not seem to be applying to our use-case.
Maybe remove it from our branch ?